src/services/print.js

```diff
@@ -291,7 +291,8 @@ ngeo.Print.prototype.encodeWmsLayer_ = function(arr, opacity, url, params) {
   const customParams = {'TRANSPARENT': true};
   if (url_url.searchParams) {
     for (const element of url_url.searchParams) {
-      customParams[element[0]] = element[1];
+      // work-around to normalize IE11 & Edge ogcserver encoding issue
+      element[0] === 'ogcserver' ? customParams[element[0]] = element[1].replace('+', ' ') : customParams[element[0]] = element[1];
```

Comment:
`Main PNG` is something we define in the Admin interface. What if the name og the ogcserver is: `Mein schönes OGC server für "PNG" & nicht für "JPEG" !`. (Worst name I can find).

I means, that perhaps we should escape more. Can't you have a try ?